orion-ui/src/maps/uiSettings.ts

```diff
@@ -0,0 +1,15 @@
+import { MapFunction } from '@/services/mapper'
+import { Settings } from '@/services/uiSettings'
+import { SettingsResponse } from '@/types/settingsResponse'
+import { FeatureFlag } from '@/utilities/permissions'
+
+export const mapSettingsResponseToSettings: MapFunction<SettingsResponse, Settings> = function(source) {
+  return {
+    apiUrl: source.api_url,
+    flags: this.map('FlagResponse', source.flags, 'Flag').filter(flagIsDefined),
+  }
+}
+
+function flagIsDefined(flag: FeatureFlag | null): flag is FeatureFlag {
```

Comment:
This would be a nice guard to use generically. Mind using this version and moving it to a type or utility file? 

```suggestion
function isNotNull<T>(value: T | null): flag is T {
```

Reply:
part 1:https://github.com/PrefectHQ/orion-design/pull/975

Reply:
part 2:https://github.com/PrefectHQ/prefect/pull/7994